kolibri/core/hooks.py

```diff
@@ -0,0 +1,40 @@
+"""
+Kolibri Core hooks
+------------------
+
+WIP! Many applications are supposed to live inside the core namespace to make
+it explicit that they are part of the core.
+
+Do we put all their hooks in one module or should each app have its own hooks
+module?
```

Comment:
I've even encountered this on the frontend-arch-feature branch -- must we enable 
"core plugins", e.g. core parts of the application that nonetheless use the plugin api?